README.md

```diff
@@ -144,7 +144,8 @@ Default value:
     "tests/**/*.rs",
     "features/**/*.rs",
     "features/**/*.rb",
-    "*specs*/**/.cs"
+    "*step*/**/*.cs",
```

Comment:
Unfamiliar with convention for placing step definitions in SpecFlow. Are there any examples, documentation or configuration from other tools you could provide that support this pattern? I would also note that this would diverge slightly from existing configuration - which might be fine; where each glue code pattern has an equivalent feature pattern, promoting that they are placed alongside each other i.e. an associated `"*step*/**/*.feature"` pattern is currently unsupported.

For reference, some examples I have located include:

- [SpecFlow Tools Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=amillard98.specflow-tools) - which checks [all `.cs` and `.feature.cs` files](https://github.com/09millarda/specflow-binder/blob/e1732f441dc209bce4ad096f8023ef95b5f74382/src/SpecflowBindingsHandler.ts#L28) - though that may be too wide a search for this extension.
- [SpecFlow-Examples repository](https://github.com/SpecFlowOSS/SpecFlow-Examples) - which keeps step definitions in directories called `StepDefinitions` or `Steps`; or containing `.Spec`
- Specflow ‘[Generating Skeleton Code](https://docs.specflow.org/projects/specflow/en/latest/visualstudio/Generating-Skeleton-Code.html)’ example - which uses ‘StepDefinitions’ as directory name

Would be great to know how the [SpecFlow Visual Studio extension](https://github.com/SpecFlowOSS/SpecFlow.VS) detects step definitions for reference - haven't yet located the relevant code.